cloudcheckr_role/main.tf

```diff
@@ -0,0 +1,351 @@
+resource "aws_iam_role_policy" "cloudcheckr_policy" {
```

Comment:
good point. that's what required from cloudcheckr. it indeed looks similar to the readonly policy but I don't think that the readonly has all the permissions listed there.